test/evergreen.yml

```diff
@@ -3636,67 +3637,86 @@ tasks:
         vars:
-          perf-test-name: 500m-btree-50r50u-backup
+          perf-test-name: 500m-btree-50r50u-backup.wtperf
           maxruns: 1
           no_create: true
           wtarg: -args ['"-C create,statistics=(fast),statistics_log=(json,wait=1,sources=[file:])"'] -ops ['"read", "update", "warnings", "max_latency_read_update"']
       - func: "upload-perf-test-stats"
         vars:
-          perf-test-name: 500m-btree-50r50u-backup
+          perf-test-name: 500m-btree-50r50u-backup.wtperf
       - func: "run-perf-test"
         vars:
-          perf-test-name: 500m-btree-80r20u
+          perf-test-name: 500m-btree-80r20u.wtperf
           maxruns: 1
           no_create: true
           wtarg: -args ['"-C create,statistics=(fast),statistics_log=(json,wait=1,sources=[file:])"'] -ops ['"read", "update", "warnings", "max_latency_read_update"']
       - func: "upload-perf-test-stats"
         vars:
-          perf-test-name: 500m-btree-80r20u
+          perf-test-name: 500m-btree-80r20u.wtperf
       - func: "run-perf-test"
         vars:
-          perf-test-name: 500m-btree-rdonly
+          perf-test-name: 500m-btree-rdonly.wtperf
           maxruns: 1
           no_create: true
           wtarg: -args ['"-C create,statistics=(fast),statistics_log=(json,wait=1,sources=[file:])"'] -ops ['"read", "warnings", "max_latency_read_update", "min_read_throughput", "max_read_throughput"']
       - func: "upload-perf-test-stats"
         vars:
-          perf-test-name: 500m-btree-rdonly
+          perf-test-name: 500m-btree-rdonly.wtperf
 
   - name: perf-test-long-checkpoint-stress
     tags: ["long-perf"]
     depends_on:
       - name: compile
     commands:
-      - command: timeout.update
-        params:
-          exec_timeout_secs: 86400
-          timeout_secs: 86400
       - func: "fetch artifacts"
       - func: "run-perf-test"
         vars:
-          perf-test-name: checkpoint-stress
+          perf-test-name: checkpoint-stress.wtperf
           maxruns: 1
           wtarg: -args ['"-C create,statistics=(fast),statistics_log=(json,wait=1,sources=[file:])"'] -ops ['"update", "checkpoint"']
       - func: "upload-perf-test-stats"
         vars:
-          perf-test-name: checkpoint-stress
+          perf-test-name: checkpoint-stress.wtperf
+
+  - name: many-dhandle-stress
+    commands:
+    depends_on:
+      - name: compile
+    commands:
+      - func: "fetch artifacts"
+      - func: "run-perf-test"
+        vars:
+          test_type: workgen
+          exec_path: ${python_binary}
+          perf-test-path: ../../../bench/workgen/runner
+          perf-test-name: many-dhandle-stress.py
+          maxruns: 1
+          wtarg: -ops ['"warning_operations", "warning_read", "warning_insert", "warning_update" ,"warning_idle", "warning_idle_create","warning_idle_drop", "max_latency_drop_diff","max_latency_drop", "max_latency_create", "max_latency_read_micro_sec", "max_latency_insert_micro_sec", "max_latency_update_micro_sec"']
```

Comment:
Can we alphabetize this list and fix the syntax? There are a few places where you put the space before the ",", example:
` ,"warning_idle"`

Reply:
sure thing, thanks.